wg-meetings/2016-10-05.md

```diff
@@ -0,0 +1,207 @@
+# Node.js Diag WG Meeting
+
+* Date: 2016-10-05
+
+## Attendees
+
+* Josh Gavant (@joshgav)
+* Thomas Watson (@watson)
+* Matt Loring    (@mattloring)
+* Kelvin Jin (@kjin000)
+* Andreas Madsen (@andreasmadsen)
+* Michael Dawson (@mhdawson)
+* Daniel Khan (@danielkhan)
+
+
+# Agenda
+
+## Previous Meeting
+
+### async_hooks
+
+Next steps:
+* One significant change remains to enable multiple listeners.
```

Comment:
Oops, actually these lines come from "Next Steps" in https://github.com/nodejs/diagnostics/blob/master/wg-meetings/2016-07-13.md so we better leave them in. Okay?